integration/nri_test.go

```diff
@@ -442,6 +445,7 @@ func TestNriLinuxCpusetAdjustmentUpdate(t *testing.T) {
 			`prev=""
              while true; do
                  cpus="$(grep Cpus_allowed_list: /proc/self/status)"
+                 echo "$cpus"
```

Comment:
Why were these changes added?

Reply:
Sorry, this is the log of my debugging. This is where I modified the omissions. I will modify it again.